types/dompurify/index.d.ts

```diff
@@ -1,87 +1,95 @@
-// Type definitions for DOM Purify
+// Type definitions for DOM Purify 2.0.1
```

Comment:
Should just be 2.0. The patch version here is updated by us and is not meant to match the original package. Please set enable the `dt-header` rule in `tslint.json`.

Reply:
Done :)

Should I also commit the `dt-header` flag set to enabled in the `tslint.json` as well?
That will cause a lint failure due to the `Definitions by` containing a non-Github url